R/get.variable.names.r

```diff
@@ -0,0 +1,155 @@
+
+
+#' This function allows to build a data table containing the original PSID variable names (of type AB12345) for each desired year and checks
+#' on the psidonline web site whether a variable is available or not. Information is obtained on https://simba.isr.umich.edu/VS/s.aspx
+#' through the package XML.
+#'
+#' Instead of building the data table by hand and look up each variable on the online dictionary to understand in which year it is available, 
+#' it is sufficient to look at the dictionary for a given wave, select some variables of interest and type in their original name for a given 
+#' year y, as well as a list of years that one wants to be included in the panel. If the variable is not available for a given year, the 
+#' function automatically types NA in the corresponding cell.
+#'
+#' Note that, in the list of variables provided, one could include a variable from year y and a variable from year y'. This is allowed and 
```

Comment:
isnt' that the main functionality of this package, i.e. you select variable names from different waves to then build a panel? not sure i understand this comment.